api/src/main/java/org/openmrs/module/fhirExtension/service/ObsBasedDiagnosticReportService.java

```diff
@@ -52,15 +52,19 @@ public class ObsBasedDiagnosticReportService extends BaseFhirService<DiagnosticR
 	
 	@Autowired
 	private DiagnosticReportObsValidator diagnosticReportObsValidator;
-
+	
 	@Autowired
 	private DiagnosticReportRequestValidator diagnosticReportRequestValidator;
 	
+	@Autowired
+	private OrderUpdateService orderUpdateService;
```

Comment:
Where this orderUpdateService is used.

Reply:
orderUpdateService is being used in create method of ObsBasedDiagnosticReportService class